data/wanderer/wanderers start.txt

```diff
@@ -1978,10 +1986,12 @@ mission "Wanderers Hai Assistance 2"
 	on offer
 		log "Traveled to the Hai homeworld with Ambassador Sayari to see if she can convince the Hai to take action to restrain the Unfettered from attacking the Wanderers. The most that the Hai were willing to do was to mass a large war fleet near Unfettered space so that the Unfettered have to divert warships there to defend their home worlds."
 		conversation
-			`The Hai are ruled by an elected "council of elders" who agree to meet with you and Sayari. After you describe the situation with the Wanderers, one of the elders says, "We had heard rumors of this. The Unfettered have recruited thousands of new warriors by telling stories of planets for us to explore and colonize, and a cowardly and foolish enemy to defeat."`
+			`The Hai's recently elected Council of "Elders" agrees to meet with you and Sayari, and the two of you make an effort to avoid directing yourselves towards Teeneep. As Sayari had quietly informed you beforehand, your young friend's popularity as an Elder does not extend to inside these walls. Apparently, Teeneep has not explained the nature of her sympathies towards the Unfettered, even to her oldest friends. It would be detrimental to your cause to specifically seek her assent here, on <origin>, for this matter. After you describe the situation with the Wanderers, one of the other elders who seems older and authoritative says, "We had heard rumors of this. The Unfettered have recruited thousands of new warriors by telling stories of planets for us to explore and colonize, and a cowardly and foolish enemy to defeat."`
```

Comment:
No 'us' is correct from the perspective that they view themselves and the Hai as one people.
Even from the perspective of the speaker, 'us' is the precise term - because the propaganda is that the opportunities are for "us" (the Hai) to explore, *by* joining the UHai.

Reply:
Yeah, that makes sense. I interpreted that as telling stories of planets for _those new warriors_ to conquer.